### PR TITLE
Update Harbor chart to enable pluggable scanner

### PR DESCRIPTION
**Pull request by ywk253100**
Update Harbor chart to enable pluggable scanner

Signed-off-by: Wenkai Yin <yinw@vmware.com>

### CODE REVIEW

**Comment by reasonerjt** [comment]
Now that clair-adapter and clair is in one pod, the service should only expose clair-adapter's API?

File: templates/core/core-cm.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ data:
   CLAIR_DB: "{{ template "harbor.database.clairDatabase" . }}"
   CLAIR_DB_SSLMODE: "{{ template "harbor.database.sslmode" . }}"
   CLAIR_URL: "http://{{ template "harbor.fullname" . }}-clair:6060"
+  CLAIR_ADAPTER_URL: "http://{{ template "harbor.clair" . }}:8080"
```
Comment (reasonerjt) [inline templates/core/core-cm.yaml]: We should only need clair_adapter_url in core's CM?

File: templates/clair/clair-dpl.yaml [hunk 2]
```diff
@@ -53,16 +53,41 @@ spec:
...
+          periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /probe/ready
+            port: 8080
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        env:
+        - name: SCANNER_CLAIR_URL
+          # To avoid a pod cannot reach itself via service IP when the clusters disable hairpin
```
Comment (reasonerjt) [inline templates/clair/clair-dpl.yaml]: I think it's totally fine as they are always in the same pod and share the same network.

File: templates/core/core-cm.yaml (hunk 1 above)
Comment (ywk253100) [reply templates/core/core-cm.yaml]: We'll remove this variables after they are deleted from main repository

**Pull request merged**